caffeine-addictt/ReFibre
# Added base features to project

## What type of PR is this? (check all applicable)




## Description
Added rootpath, base template, navigation bar and a footer




## Related Tickets & Documents



- Related Issue #
- Closes #



## QA Instructions, Screenshots, Recordings

Im not sure if i did add the rootpath, but mostly to test it out i guess just run the code? to ensure that the navbar and footer work, though they dont have any links to them yet, it is just to see if they look ok.



## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_



## [optional] Are there any post deployment tasks we need to perform?




## Checklist
